src/libraries/System.Linq.Expressions/tests/Default/DefaultTests.cs

```diff
@@ -106,5 +106,50 @@ private struct StPri
         {
             public int IntProperty { get; set; }
         }
+
+        [Theory, ClassData(typeof(CompilationTypes))]
+        public void StructTypeWithParameterlessConstructor(bool useInterpreter)
+        {
+            Expression<Func<ValueTypeWithParameterlessConstructor>> lambda =
+                Expression.Lambda<Func<ValueTypeWithParameterlessConstructor>>(
+                    Expression.Default(typeof(ValueTypeWithParameterlessConstructor)));
+            Func<ValueTypeWithParameterlessConstructor> func = lambda.Compile(useInterpreter);
+            ValueTypeWithParameterlessConstructor defaultValue = func();
+            Assert.False(defaultValue.ConstructorWasRun);
+        }
+
+        [Theory, ClassData(typeof(CompilationTypes))]
+        public void StructTypeWithParameterlessConstructorThatThrows(bool useInterpreter)
+        {
+            Expression<Func<ValueTypeWithParameterlessConstructorThatThrows>> lambda =
+                Expression.Lambda<Func<ValueTypeWithParameterlessConstructorThatThrows>>(
+                    Expression.Default(typeof(ValueTypeWithParameterlessConstructorThatThrows)));
+            Func<ValueTypeWithParameterlessConstructorThatThrows> func = lambda.Compile(useInterpreter);
+            ValueTypeWithParameterlessConstructorThatThrows defaultValue = func();
+            Assert.Null(defaultValue.Value);
+        }
+
+        [Theory, ClassData(typeof(CompilationTypes))]
+        public void NullableStructTypeWithParameterlessConstructor(bool useInterpreter)
+        {
+            Expression<Func<ValueTypeWithParameterlessConstructorThatThrows?>> lambda =
+                Expression.Lambda<Func<ValueTypeWithParameterlessConstructorThatThrows?>>(
+                    Expression.Default(typeof(ValueTypeWithParameterlessConstructorThatThrows?)));
+            Func<ValueTypeWithParameterlessConstructorThatThrows?> func = lambda.Compile(useInterpreter);
+            ValueTypeWithParameterlessConstructorThatThrows? defaultValue = func();
+            Assert.Null(defaultValue);
+        }
+
+        [Theory]
+        [ClassData(typeof(CompilationTypes))]
+        public static void GetValueOrDefault(bool useInterpreter)
```

Comment:
I didn't understand whether there was significance to this being `static`.

Reply:
It looks like we have a mix of `static` and instance tests. The only reason this one is `static` is it was originally added in a location where others were `static`. Removed the `static` here for consistency.